_posts/2018-10-02-release-of-text2num-1.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "Text2num version 1.0.0 released!"
+excerpt: "There already existed some python packages to convert numbers written in english into python numbers or their decimal
+digit representation, but there was nothing available for the french language. That's why we developped this library and shared it with the community."
+date:   2018-10-02 12:53:00 +0100
+categories: python
+tags: python NLP
+---
+
+The output of speech-to-text systems are entirely made of words, without punctuation or capitalization. This makes visual scanning for numbers quite cumbersome,
+especially in transcriptions of real life dialogues as they also contain a lot of gibberish words — like « heu, ben, bah… » — and the syntax or
+grammar is not always right. Besides that, text mining tools and techniques often, if not always, expect numbers to be in decimal digit representation.
+
+That's why we decided to add a transformation pass to our speech-to-text engine in order to convert all spoken numbers into their digit spelling.
+
+Here at Allo-Media, we are fond of Open Source Software. So we first looked at the state of the art of python libraries for parsing words into numbers. There were some for the english language, but we didn't find any for french. Therefore, we decided to build our own library and contribute it back to the community.
+
+Here are two samples of what you can expect from it:
+
+```python
+>>> from text_to_num import text2num
+>>> text2num('nonante-cinq')
```

Comment:
Maybe adding the classical 'quatre-vingt quinze' before would avoid confusion like: "This is a belgian parser!" :)